packages/playback-core/src/index.ts

```diff
@@ -251,18 +253,20 @@ export const loadMedia = (
   props: Partial<
     Pick<
       MuxMediaProps,
-      "preferMse" | "src" | "type" | "startTime" | "streamType"
+      "preferMse" | "src" | "type" | "startTime" | "streamType" | "autoplay"
```

Comment:
blocking: We need to account for and validate against React for the camelcase of `autoPlay`. Should be able to test against `<MuxVideo/>` & `<MuxAudio/>` (React). As a quick fix we can just allow both here.

Reply:
Seems to work without any type errors. Probably by the time it gets here from react, it gets renamed to `autoplay`. Also, locally, both `autoplay` and `autoPlay` properties work in React.
https://elements-demo-nextjs-git-fork-gkatsev-autoplay-shenanigans-mux.vercel.app/MuxVideo